pkg/metrics/driver/registry/registry.go

```diff
@@ -0,0 +1,59 @@
+// Copyright 2018-2020 CERN
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// In applying this license, CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+package registry
+
+import (
+	"os"
+
+	"github.com/cs3org/reva/pkg/logger"
+	"github.com/cs3org/reva/pkg/metrics/reader"
+	"github.com/rs/zerolog"
+)
+
+var log zerolog.Logger
+
+var drivers map[string]reader.Reader // map key is driver type name
+
+func init() {
```

Comment:
Remove the logs. Let's keep it simple https://github.com/cs3org/reva/blob/master/pkg/ocm/invite/manager/registry/registry.go

Reply:
Ok, I do the loading like in that registry as well. That is better.
Do you agree with the GetDriver() method or do you want me to 'copy' https://github.com/cs3org/reva/blob/master/pkg/ocm/invite/manager/registry/registry.go ?